docs/google/engine.md

```diff
@@ -191,3 +191,56 @@ for b in range(num_circuits_in_batch):
      idx+=1
 ```
 
+
+## Downloading historical results
+
+Results from  previous computations are archived and can be downloaded later
+by those in the same cloud project.  You must use the same project id to
+access historical results or your request will be denied.
+In order to retrieve previous results, you will need the program id and the
+job id.  This can be retrieved from the job object when you run a sweep.
+Currently, getting the program and job ids can only be done through the
```

Comment:
I realize that this is a documentation PR, but still think it's worth calling that that we really should have "list" methods as well, but they don't exist in Cirq today.

Reply:
Sure.  Want to create a bug for it and I'll go ahead and assign it to me?

Reply:
Added https://github.com/quantumlib/Cirq/issues/3302